readme.md

```diff
@@ -83,13 +83,10 @@ the remaining content on either end, if any, is removed.
 #### Trim Algorithm
```

Comment:
Should we use "long long" for sizes like Blob does in the FileAPI W3c spec?

https://www.w3.org/TR/FileAPI/#idl-index

Reply:
Yes. We should. I will update these.

Reply:
Updated.